RaspberryPiDeployment.md

```diff
@@ -13,7 +13,7 @@ The process is basically the following:
 
 BTCPayServer can be successfully installed on the following hardware.   
  
-1. Raspberry Pi 3 Model B+
+1. Raspberry Pi 3 Model B+, Raspiberry Pi 4 Model B
```

Comment:
I think what we're going to do is make a separate document for RPI4 because the instructions have some significant differences

Reply:
Ok. It may not even be worth mentioning the Raspberry Pi 3 anymore.  BTCPayServer can be installed on it but memory is an issue.  BTCPayServer works great 2GB but obviously 4GB is better.;